public/locales/es/dashboard.json

```diff
@@ -4,6 +4,7 @@
   },
   "title": "Tus noticias",
   "moduleMap": "Mapa de módulos",
+  "back-to-dashboard": "Volver a dashboard",
```

Comment:
Use the one we are using in the cohort dashboard that says Back to choose program and change the translation to Back to dashboard